services/embedding/server.py

```
@@ -0,0 +1,219 @@
+
+        try:
+            loop = asyncio.get_running_loop()
+            start = time.time()
+            vectors = await loop.run_in_executor(
+                None,
+                lambda: self._embedder.embed_texts(
+                    list(request.texts), task=task, batch_size=batch_size
+                ),
+            )
+            duration_ms = int((time.time() - start) * 1000)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)